tests/factories/test_factory.py

```diff
@@ -62,33 +62,6 @@ def test_resource_class_create():
     assert rec_type.resource_config_cls.url_prefix == "/resourcetests"
     assert rec_type.resource_config_cls.blueprint_name == "resourcetest"
 
-    # links_schema_class = rec_type.resource_config_cls.links_config["record"]
```

Comment:
below tests that links_item and links_search setup. Then, all tests that use the service validate that the defined links work (they fail in response serialization like `.data` otherwise).